.github/dependabot.yml

```diff
@@ -31,17 +31,9 @@ updates:
       interval: weekly
 
   - directory: /
-    # ignore:
-    #   # Managed by cisagov/skeleton-tf-module
-    #   - dependency-name: hashicorp/aws
-    package-ecosystem: terraform
-    schedule:
-      interval: weekly
-
-  - directory: /examples/basic_usage
-    # ignore:
-    #   # Managed by cisagov/skeleton-tf-module
-    #   - dependency-name: hashicorp/aws
+    ignore:
+      # Managed by cisagov/skeleton-tf-module
+      - dependency-name: hashicorp/aws
     package-ecosystem: terraform
```

Comment:
Is this skeleton meant to be a (lineage) descendant of `skeleton-tf-module`?  If so, you are missing the `.github/lineage.yml` file.  If not, then I don't think we should be ignoring this dependency here.

Reply:
Aha, I forgot about that; I was only looking at the code changes in the PR. 👍